index.js

```diff
@@ -1,10 +1,44 @@
 const babel = require('babel-core');
+const fs = require('fs');
+const path = require('path');
 
-const babelOpts = {
-  plugins: ['transform-flow-strip-types', 'syntax-jsx']
+/**
+ * Remove duplicates from array
+ * @param  {array} items Input array
+ * @return {array} Output array with unique values
+ */
+const arrayUnique = (items) =>
+    items.filter((item, pos, self) => self.indexOf(item) === pos);
+
+/**
+ * Construct Babel plugins config object.
+ * Honours user's global Babel plugins config. and adds
+ * required Flowtype plugin if not already available.
+ *
+ * @return {Object}
+ */
+const buildBabelrcConfig = () => {
+  const defaultConfig = ['transform-flow-strip-types', 'syntax-jsx'];
+  const babelrcPath = path.normalize(path.join(__dirname, '..', '..', '.babelrc'));
+
+  if (! fs.existsSync(babelrcPath)) {
+    return { plugins: defaultConfig };
+  }
+
+  const babelrc = JSON.parse(fs.readFileSync(babelrcPath));
```

Comment:
yup. good shout, added encoding. 
Although, I retained the sync file loading as this enough for the needs here. :)